.github/workflows/rust-release-to-crates.yml

```
@@ -0,0 +1,32 @@
+name: Rust release to crates.io
```

Rename this to `rust-publish`. We use the "release" terminology separately.

```suggestion
name: Rust public to crates.io
```